addon/popup.html

```diff
@@ -112,6 +112,10 @@
     </div>
     <button id="reload">Reload rules</button>
     <button id="reset">Reset settings</button>
+    <div>
+      <input type="checkbox" id="filterlist" name="filterlist" checked>
+      <label for="filterlist">Enable cosmetic filterlist</label>
```

Comment:
Yep, this is internal, and I'm not sure we will actually merge this to main

Reply:
OK, if it stays internal I guess that's OK, though maybe showing more context would make it easier to use/understand. But if this does go out to the public, I think proper attribution and explanation should definitely be given. This might be something to check with folks internally if you didn't already.